axiom/nr_errors.c

```diff
@@ -94,6 +153,10 @@ void nr_error_destroy(nr_error_t** error_ptr) {
   nr_free(error->klass);
   nr_free(error->span_id);
   nr_free(error->stacktrace_json);
+  if ((NULL != error->error_file) && (NULL != error->error_context)) {
```

Comment:
`nr_free` should handle this case.

Reply:
Fixed: [c058df8](https://github.com/newrelic/newrelic-php-agent/pull/942/commits/c058df86277f602e0f73e37e0a0a86e805a2d7d4)